discovery-infra/tests/base_test.py

```diff
@@ -180,6 +183,19 @@ def _collect_virsh_logs(self, nodes, log_dir_name):
         infra_utils.run_command(f"journalctl --since \"{nodes.setup_time}\" "
                                 f"-u libvirtd -D /run/log/journal >> {libvird_log_path}", shell=True)
 
+    def _collect_journalctl(self, nodes, log_dir_name):
+        logging.info('Collecting journalctl\n')
+        infra_utils.recreate_folder(log_dir_name, with_chmod=False ,force_recreate=False)
+        journal_ctl_path = Path(log_dir_name) / 'nodes_journalctl'
+        infra_utils.recreate_folder(journal_ctl_path, with_chmod=False)
+        for node in nodes:
+            try:
+                node.run_command(f'sudo journalctl >> /tmp/{node.name}-journalctl')
+                journal_path = journal_ctl_path / node.name
+                node.download_file(f'/tmp/{node.name}-journalctl', journal_path)
```

Comment:
I think this should be `str(journal_path)` since `download_file` expects `str` and not a `Path` object. 
Sorry for making everything complex.. :)

Reply:
hmm, not sure how this worked

changing